### ISSUE TITLE
Font size error

### ISSUE THREAD

**Issue by ikbalyasar** [hsylife/SwiftyPickerPopover#103]
StringPickerPopover value, I assign does not change.

**Comment by hsylife** [comment]
Hello.
Could you share some code?

**Comment by ikbalyasar** [comment]
Hi,
 ```
let p = StringPickerPopover(title: "xyz", choices: MyArray)
            .setFontSize(3.0)  // Dont work
```

**Comment by hsylife** [comment]
I will inspect it tonight!

**Comment by hsylife** [comment]
The fixed version(6.6.3) has been released. Please `pod update`.
